sample-lnd.conf

```diff
@@ -0,0 +1,180 @@
+[Application Options]
+
+; The directory to store lnd's data within
+datadir=~/.lnd/data
+
+; Path to TLS certificate for lnd's RPC and REST services
+tlscertpath=~/.lnd/tls.cert
+
+; Path to TLS private key for lnd's RPC and REST services
+tlskeypath=~/.lnd/tls.key
+
+; Disable macaroon authentication
+no-macaroons=false
+
+; Path to write the admin macaroon for lnd's RPC and REST services if it 
+; doesn't exist
+adminmacaroonpath=~/.lnd/admin.macaroon
+
+; Path to write the read-only macaroon for lnd's RPC and REST services if it
+; doesn't exist
+readonlymacaroonpath=~/.lnd/readonly.macaroon
+                       
+; Directory to log output.
+logdir=~/.lnd/logs
+
+; Add an interface/port to listen for connections 
+; (default all interfaces port: 9735)
+listen=
+
+; Add an ip to the list of local addresses we claim to listen on to peers
+;externalip=            
+
+
+; Logging level for all subsystems {trace, debug, info, warn, error, critical} --
+; You may also specify <subsystem>=<level>,<subsystem2>=<level>,... to set the
+; log level for individual subsystems -- Use show to list available subsystems
+debuglevel=info
+
+; Write CPU profile to the specified file
+;cpuprofile=
+
+; Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65536
+;profile=
+
+; The port to listen on for incoming p2p connections
+peerport=9735
+
+; The port for the rpc server
+rpcport=10009
+
+; The port for the REST server
+restport=8080
+
+; Activate the debug htlc mode. With the debug HTLC mode, all payments sent use a
+; pre-determined R-Hash. Additionally, all HTLCs sent to a node with the debug
+; HTLC R-Hash are immediately settled in the next available state transition.
+debughtlc=false
+
+; Activate the hodl HTLC mode.  With hodl HTLC mode, all incoming HTLCs will be
+; accepted by the receiving node, but no attempt will be made to settle the
+; payment with the sender.
+hodlhtlc=false
```

Comment:
A comment should be added to detail that this option and the one above it exist primarily for testing purposes. 

In the case of `debughtlc` it uses the _same_ payment hash _each time_. This means that intermediate nodes that see this payment hash can instantly take the funds locked within the output during the clearing phase, skipping forwarding the HTLc to any other participants.

Reply:
Maybe we should just skip the ones meant for testing? Testers know them anyway and it can only be confusing and take up space.

Reply:
I would remove 
- debughtlc
- hodlhtlc
- simnet
- regtest
to be honest.

Reply:
`simnet` and `regtest` are fine as developers may be using that locally for testing purposes. It's just the two HTLC modifiers that are only used within our automated integration tests